### ISSUE TITLE
Sqlboiler doesn't work with variable originated from c.Query in gin framework

### ISSUE THREAD

**Issue by jack-spar** [volatiletech/sqlboiler#1270]
If you're having a generation problem please answer these questions before submitting your issue. Thanks!

### What version of SQLBoiler are you using (`sqlboiler --version`)?
SQLBoiler v4.14.1

### What is your database and version (eg. Postgresql 10)
mysql  Ver 8.0.32

### Further information. What did you do, what did you expect?
I want to use sqlboiler with a variable which come from c.Query() in gin framework, but sqlboiler returns ERROR: `sql: no rows in result set`. Are there any ways to pass the variable? Here is my sample code:

```golang
func getUser(c *gin.Context){
    name := c.Query("name")
    /* some db setting */
    user, err := models.Users(
        qm.WhereIn("name in ?", name),
    ).One(ctx, db)
}
```
I know that this works:
```golang
func getUser(){
    name := "John"
    /* some db setting */
    user, err := models.Users(
        qm.WhereIn("name in ?", name),
    ).One(ctx, db)
}
```

**Comment by stephenafamo** [comment]
The error `sql: no rows in result set` means that the query returned no matching rows. This is not unique to sqlboiler or gin.

For example, you would get the same doing this:

```go
var name = ""
userRow, err := db.QueryRowContext(ctx, "SELECT * FROM users WHERE name = ?", name)
// err will likely be sql.ErrNoRows
```

The typical way to ignore this error would be to do something like this:

```go
if err != nil && !errors.Is(err, sql.ErrNoRows) {
  return err
}
```

Note that if you ignore `sql.ErrNoRows`, then the `user` may be `nil`.

**Comment by jack-spar** [comment]
Thank you for your reply. Because I don't have permission to reopen this issue, I opened here: https://github.com/volatiletech/sqlboiler/issues/1271.
My problem is where `ERROR: sql: no rows in result set.` arise even though I can get a params through gin.
I exemplify the problem at the code in #1271 .

**Comment by jack-spar** [comment]
I noticed that the reason why `sql: no rows in result set` occurs is failure in my get request.
I called request in next format: `http://localhost:8080/api/user?name="John"`.
However, right request format is here: `http://localhost:8080/api/user?name=John`.
Just, I remove double quotes, and sqlboiler can search `John`, not `"John"`.

I'm sorry for a easy and basic mistake for GET request.
There is no problem in both gin or sqlboiler. It's just my silly mistake.
Anyway, thank you for your clear and coherent advise.
